web/packages/teleport/src/features.tsx

```diff
@@ -97,6 +100,30 @@ export class FeatureNodes implements TeleportFeature {
   }
 }
 
+export class FeatureUnifiedResources implements TeleportFeature {
+  route = {
+    title: 'Resources',
+    path: cfg.routes.resources,
+    exact: true,
+    component: UnifiedResources,
+  };
+
+  navigationItem = {
+    title: 'Resources',
+    icon: <ServersIcon />,
+    exact: true,
+    getLink(clusterId: string) {
+      return cfg.getUnifiedResourcesRoute(clusterId);
+    },
+  };
+
+  category = NavigationCategory.Resources;
+
+  hasAccess() {
+    return localStorage.areUnifiedResourcesEnabled();
+  }
```

Comment:
hrm...i'm not sure how i see local storage being used to set the flag in the future, so if it was a temporary thing i'd remove the local storage changes and manually change flag as you need like this:

```suggestion
  hasAccess() {
    return false // TODO(bl-nero) etc
  }
```

or like this
```
hasAccess(flags: FeatureFlags) {
    return flags.unifiedResources;  // not sure if we ever NOT want to render it though
}
```

and define the flag here: https://github.com/gravitational/teleport/blob/6fb9f081084fda77f1ab0b93834d20efcb843e0c/web/packages/teleport/src/teleportContext.tsx#L95

Reply:
Actually, I found a better way to do it: instead of using `hasAccess`(), I simply don't register the feature in the first place in `getOSSFeatures()`.